src/app/features/account/components/password-forgot/password-forgot.component.html

```diff
@@ -0,0 +1,33 @@
+<ng-container *ngIf="!forgotPasswordFormSubmitted else mailSent">
+  <form (ngSubmit)="onSubmit()" [formGroup]="forgotPasswordForm" class="app-page-section app-form--simple">
+    <h1>{{ 'ACCOUNT_FEATURE.FORGOT_PASSWORD.PAGE_TITLE' | translate }}</h1>
+
+    <app-form-alerts [alerts]="forgotPasswordFormAlerts">
+    </app-form-alerts>
```

Comment:
`app-form-alerts` and `h1` in Estate are outside the form because we don't want to follow `app-form--simple` layout for them